src/main/java/org/stellar/sdk/CreatePassiveOfferOperation.java

```diff
@@ -1,135 +1,13 @@
 package org.stellar.sdk;
 
-import org.stellar.sdk.xdr.CreatePassiveOfferOp;
-import org.stellar.sdk.xdr.Int64;
-import org.stellar.sdk.xdr.OperationType;
-
-import java.math.BigDecimal;
-import java.math.MathContext;
-
-import static com.google.common.base.Preconditions.checkNotNull;
-
 /**
- * Represents <a href="https://www.stellar.org/developers/learn/concepts/list-of-operations.html#create-passive-offer" target="_blank">CreatePassiveOffer</a> operation.
- * @see <a href="https://www.stellar.org/developers/learn/concepts/list-of-operations.html" target="_blank">List of Operations</a>
+ * @deprecated Use {@link CreatePassiveSellOfferOperation}
```

Comment:
Should I deprecate a class like this?

Reply:
We should deprecate this class but it needs to work until removed. We are still pre 1.0.0 version so we can remove it in the next minor version (assuming these changes will be part of 0.7.0, then deprecated methods can be removed in 0.8.0).

Reply:
I've added them back, but some behavior may change due to the XDR changes.